pkg/cmd/release/config.go

```diff
@@ -0,0 +1,101 @@
+package release
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+	"github.com/openshift/ci-tools/pkg/registry"
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+type configOptions struct {
+	*options
+	list, resolve bool
+}
+
+func newConfigCommand(o *options) *cobra.Command {
+	co := configOptions{options: o}
+	ret := &cobra.Command{
+		Use:   "config",
+		Short: "ci-operator configuration commands",
+		Long: `Loads configuration files and writes them to stdout, optionally performing
+resolution.`,
+		RunE: func(_ *cobra.Command, args []string) error {
```

Comment:
`release config --config-dir=$RELEASE_REPO/ci-operator/config/openshift ci-tools api`
works fine, but
`release config -l --config-dir=$RELEASE_REPO/ci-operator/config/openshift ci-tools api` doesn't.
It somehow breaks what it has would have been a nice symmetry.

Reply:
Nice, fixed.